servicebus/data-plane/azure-servicebus/src/main/java/com/microsoft/azure/servicebus/management/SubscriptionDescription.java

```diff
@@ -137,14 +134,13 @@ public Duration getDefaultMessageTimeToLive() {
      * See {@link #getDefaultMessageTimeToLive()}
      */
     public void setDefaultMessageTimeToLive(Duration defaultMessageTimeToLive) {
-        if (defaultMessageTimeToLive != null &&
-                (defaultMessageTimeToLive.compareTo(ManagementClientConstants.MIN_ALLOWED_TTL) < 0 ||
-                        defaultMessageTimeToLive.compareTo(ManagementClientConstants.MAX_ALLOWED_TTL) > 0))
-        {
+        if (defaultMessageTimeToLive != null
```

Comment:
I think this encounters the same issue I pointed out in #3527. You're fixing the potential NullPointerException, but consequently, you're allowing `null` to be a valid value to pass into this method.

Reply:
Same with the others below.